content_zh/blog/2017/0.1-using-network-policy/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: 在 Istio 下使用 Network Policy
+description: Kubernetes 中的 Network Policy 如何与 Istio 策略关联。
+publishdate: 2017-08-10
+subtitle:
+attribution: Spike Curtis
+weight: 97
+aliases: 
```

Comment:
Please remove aliases, they're only used for old files moving to a new location. This is a new file.

Reply:
thx for review, aliases have been removed